src/Modal.js

```diff
@@ -49,26 +50,48 @@ const AsyncWrapper = React.createClass({
         // XXX: temporary logging to try to diagnose
         // https://github.com/vector-im/riot-web/issues/3148
         console.log('Starting load of AsyncWrapper for modal');
-        this.props.loader((e) => {
+        this.props.prom.then((result) => {
             // XXX: temporary logging to try to diagnose
             // https://github.com/vector-im/riot-web/issues/3148
-            console.log('AsyncWrapper load completed with '+e.displayName);
+            console.log('AsyncWrapper load completed with '+result.displayName);
```

Comment:
do we even need this log line anymore? The referenced issue was closed in April 2017.